packages/test-runner-core/src/config/TestRunnerCoreConfig.ts

```diff
@@ -27,6 +27,7 @@ export interface CoverageConfig {
 export interface TestRunnerCoreConfig {
   rootDir: string;
   files?: string | string[];
+  allowPassWithoutTests?: boolean;
```

Comment:
related to another question: I'd just make it `allowPassWithoutTests: boolean` and default to "false", here and below